salt/orch/update.sls

```diff
@@ -219,6 +243,17 @@ pre-orchestration-migration:
     - require:
       - salt: {{ worker_id }}-update-post-start-services
 
+# Remove the now defuct caasp_fqdn grain
+{{ worker_id }}-remove-caasp-fqdn-grain:
+  salt.function:
+    - tgt: {{ worker_id }}
+    - name: grains.delval
+    - arg:
+      - caasp_fqdn
+      - {{ worker_id }}
+    - require:
+      - salt: {{ worker_id }}-update-reboot-needed-grain
+
```

Comment:
Maybe we can unify the `*-remove-caasp-fqdn-grain` to be a single step after the update has succeeded (at the end of the orchestration?), targeting all nodes, just to keep things simple instead of strictly in order.

Reply:
Good idea...